### ISSUE TITLE
Providing accessibility coefficients for the juvenile and adult life stages of focal functional groups

### ISSUE THREAD

**Issue by agruss2** [jhpoelen/fb-osmose-bridge#133]
For each focal functional group, two accessibility coefficients should be provided in the "predation-accessibility.csv" file: one for juveniles, and one for adults. 
To be able to implement this, the API needs to consider the following parameters from the "osm_param-predation.csv" file: (1) "predation.accessibility.stage.structure"; and (2) the "predation.accessibility.stage.threshold.spX" parameters, which are ages in years if predation.accessibility.stage.structure = age, or sizes in cm if predation.accessibility.stage.structure = size. 
Ultimately, one should obtain a "predation-accessibility.csv" file like this one: 
[predation-accessibility.zip](https://github.com/jhpoelen/fb-osmose-bridge/files/1271493/predation-accessibility.zip). (This is the "predation-accessibility.csv" file for the OSMOSE model of the West Florida Shelf). 
Note that, because the current configuration of the bridge between FishBase/SeaLifeBase and OSMOSE, the accessibility coefficients of the juvenile and adult stages of a given focal functional group will be identical. However, we want to provide two sets of accessibility coefficients (two columns in the "predation-accessibility.csv" file) for each focal functional group (one set of accessibility coefficients for juveniles, and one set of accessibility coefficients for adults), so as to give the possibility to the user to alter the values provided by the bridge between FishBase/SeaLifeBase and OSMOSE if they want. 
Many thanks.

**Comment by jhpoelen** [comment]
Please review the first pass at implementing this feature.

**Comment by jhpoelen** [comment]
I've just generated a table containing using the Iceland Shelf/Sea defaults. Part of table looks like - 


v Prey / Predator > | fishpelagicsm < 3 year | fishpelagicsm > 3 year | fishpelagicmd < 2 year | fishpelagicmd > 2 year
-- | -- | -- | -- | --
fishpelagicsm < 3 year | 0.8 | 0.8 | 0.4 | 0.4
fishpelagicsm > 3 year | 0.8 | 0.8 | 0.4 | 0.4
fishpelagicmd < 2 year | 0.4 | 0.4 | 0.8 | 0.8
fishpelagicmd > 2 year | 0.4 | 0.4 | 0.8 | 0.8
fishpelagiclg < 2.5 year | 0.4 | 0.4 | 0.4 | 0.4
fishpelagiclg > 2.5 year | 0.4 | 0.4 | 0.4 | 0.4
fishdemersalsm < 2 year | 0.4 | 0.4 | 0.4 | 0.4
fishdemersalsm > 2 year | 0.4 | 0.4 | 0.4 | 0.4
fishdemersalmd < 3 year | 0.4 | 0.4 | 0.08 | 0.08

**Comment by agruss2** [comment]
Perfect, thanks a lot! I just ran a test for the Iceland Shelf/Sea ecosystem, and the behavior now is as expected. I am therefore closing the present issue.
